test/features/associating_granules_test.rb

```diff
@@ -19,23 +19,29 @@ class AssociatingGranulesTest < Capybara::Rails::TestCase
     describe 'associate granules to collections' do
       it 'can assign granule to collection' do
         visit '/home'
-        wait_for_jQuery(5)
 
-        all('#record_id_')[0].click # Selects the first collection record in the list
-        find('#open > div > div.navigate-buttons > input').click # click the See Review Details button
-        first("#associated_granule_value").find("option[value='Undefined']").click # other tests are not cleaning up db, so reset it back manually
-        first("#associated_granule_value").find("option[value='5']").click # Click in select box for what granule to associate
+        within '#open' do
```

Comment:
All changes in this test are to use the within block which makes things easier, no real functionality changes to the tests.   Also with the additional fixtures, scoping using within helps to prevent future breakage.